Unix/doc/setup-kerberos-omi.md

```diff
@@ -0,0 +1,183 @@
+#Enabling Kerberos Authentication on OMI
```

Comment:
Need add a blank after all # or ##, so the markdown format can show correctly, thanks.